### ISSUE TITLE
re v1.106 Some scrapers not working

### ISSUE THREAD

**Issue by bjbca** [a4k-openproject/script.module.openscrapers#127]
v1.106
I dont use many free scrapers, i do use most debrid scrapers, of the scrapers i use i have had these issues, P.S. Thank you for all your efforts, im just trying to contribute:
[2020-03-17 05:37:04] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: Error: Loading module: "projectfreetv": cannot import name cfScraper[2020-03-17 05:37:06] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: Request-Error (500): http://www.sceneddl.me/?s=Riviera+S02E10
[2020-03-17 05:37:06] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: Request-Error: (unknown url type: Riviera) => Riviera
[2020-03-17 05:37:06] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: Request-Error: (unknown url type: Riviera) => Riviera
[2020-03-17 05:37:06] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: Request-Error: (unknown url type: Riviera) => Riviera
[2020-03-17 05:37:07] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: MYVIDEOLINK - Exception: 
Traceback (most recent call last):
  File "/Users/xxxxxxx/Library/Application Support/Kodi/addons/script.module.openscrapers/lib/openscrapers/sources_openscrapers/en_DebridOnly/myvideolink.py", line 107, in sources
    posts = zip(client.parseDOM(r1, 'a', ret='href'), client.parseDOM(r1, 'a'), re.findall('((?:\d+\.\d+|\d+\,\d+|\d+)\s*(?:GB|GiB|MB|MiB))', r2[0]))
[2020-03-17 05:38:03] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: RAPIDMOVIEZ - Exception: 
Traceback (most recent call last):
Cloudflare_reCaptcha_Provider: Cloudflare reCaptcha detected, unfortunately you haven't loaded an anti reCaptcha provider correctly via the 'recaptcha' parameter.
[2020-03-17 05:38:03] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: RAPIDMOVIEZ - Exception: 
Traceback (most recent call last):
MissingSchema: Invalid URL 'None': No schema supplied. Perhaps you meant http://None?
[2020-03-17 05:39:10] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: RAPIDMOVIEZ - Exception: 
Traceback (most recent call last):
Cloudflare_Loop_Protection: !!Loop Protection!! We have tried to solve 3 time(s) in a row.
[2020-03-17 05:39:10] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: RAPIDMOVIEZ - Exception: 
Traceback (most recent call last):
Cloudflare_Loop_Protection: !!Loop Protection!! We have tried to solve 3 time(s) in a row.
[2020-03-17 05:39:10] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: RAPIDMOVIEZ - Exception: 
Traceback (most recent call last):
Cloudflare_Loop_Protection: !!Loop Protection!! We have tried to solve 3 time(s) in a row.
[2020-03-17 05:39:10] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: RAPIDMOVIEZ - Exception: 
Traceback (most recent call last):
Cloudflare_Loop_Protection: !!Loop Protection!! We have tried to solve 3 time(s) in a row.
[2020-03-17 05:39:14] [COLOR red][ OPENSCRAPERS DEBUG ][/COLOR]: RAPIDMOVIEZ - Exception: 
Traceback (most recent call last):
Cloudflare_Loop_Protection: !!Loop Protection!! We have tried to solve 3 time(s) in a row.

Seems Rapidmoviez rarely passes CF...

**Comment by 123Venom** [comment]
`cannot import name cfScraper`
That jumped out at me because note the cap S, which was a modification Tikipete did that we never incorporated for now.  Not sure how you end up with that.  When I was testing Tikipete's mod it was later decided we'd hold off for now on incorporating the change so I reverted the changes to the scrapers I was working on only for debrid and torrents..didn't bother with en's cause I wasn't working on them.  Then I discovered that ganool site url changed to fmovies so I attempted to rename it, unknowingly we already had an en/fmovies.  So it attempted to also import the en version which was still using Tikipete's `cfScrape`.  I suspect it's something like that for you and you might have to do a scan to see what's calling that. I just did a full scan with case sensitivity and `cfScrape` does not exist in the v1.106 here.

ya cfscrape module still has random issues with IUAM_challenge.  I checked http://rapidmoviez.cr/ to be sure they've not changed anything and it is still using IUAM.  What I've seen with cfscrape is if server response time is slow it gets misinterpreted as `Cloudflare_Loop_Protection`.  It was working fine couple days ago so lets see if it clears up.  Pirateiro torrent scraper is notorious for this.  One minute the IUAM_challenge passes and loads results...other times log is spammed with that Loop_Protection..then it clears up again...lol.  It's got to be a timeout issue causing cfscrape to log it incorrectly.
